projects/demo/src/modules/markup/icons/icons-group/icons-group.template.html

```diff
@@ -4,33 +4,36 @@ <h2 class="title">Search by name</h2>
     tuiTextfieldSize="m"
     [tuiTextfieldLabelOutside]="true"
     [(ngModel)]="search"
```

Comment:
If we are switching to RxJS, the proper way to handle this is to use `FormControl` and pipe from `valueChanges`.

Reply:
fixed